levels/fullstack.tex

```diff
@@ -0,0 +1,251 @@
+\documentclass{article}
+
+\usepackage[table]{xcolor}
+\usepackage[paperwidth=800mm, paperheight=500mm]{geometry}
+
+% Don't show page numbers.
+\pagenumbering{gobble}
+
+% Column settings for skills able.
+\newcolumntype{C}{p{18em}}
+
+\begin{document}
+
+% Title of matrix.
+\begin{center}
+\begin{huge}
+Fullstack Engineering
+\end{huge}
+\end{center}
+
+% Alternating row colors.
+\rowcolors{2}{gray!80!gray!50}{gray!50!gray!20}
+
+% Padding for each cell.
+{\renewcommand{\arraystretch}{2}
+
+% Skills table.
+\begin{tabular}{ | C | C | C | C | C | C | C | C |}
+  \hline
+  \rowcolor{blue!60!black!10}
+    Level/Skill &
+
+    \cellcolor{blue!25}
+    \textbf{Product Output} &
+    \textbf{Data structures/Performance} &
+    \textbf{Security} &
+    \textbf{Communication/Writing} &
+    \textbf{Networking} &
+    \textbf{Fullstack Engineering} &
+    \textbf{Tooling}
+    \\
+  \hline
+    1&
+
+    \cellcolor{blue!25}
+    Collaborates with product through designs, mockups, and demos of product
+    features before implementation.
+    &
+
+    Demonstrates good understanding of basic data structures like hash tables,
+    linked lists, and trees. Can reason about algorithm complexity. Applies relevant
```

Comment:
I'm not sure how relevant the knowledge about those data structures is for devs erring on the side of web development. I realize that this might be one of the first things they teach you at a university. But from someone on this level (this is level 1 we're talking about) I'd expect just knowing the performance characteristics of arrays vs maps.

```suggestion
    Demonstrates good understanding of basic data structures. Can reason about algorithm complexity. Applies relevant
```